include/rocksdb/c.h

```diff
@@ -2420,7 +2458,7 @@ rocksdb_slicetransform_create(
     unsigned char (*in_range)(void*, const char* key, size_t length),
     const char* (*name)(void*));
 extern ROCKSDB_LIBRARY_API rocksdb_slicetransform_t*
-    rocksdb_slicetransform_create_fixed_prefix(size_t);
+rocksdb_slicetransform_create_fixed_prefix(size_t);
```

Comment:
Unrelated, but `clang-format` insists, and eventually I gave up fighting it.

Reply:
Perhaps revert it before merge, we don't want to create more reasons for rebase conflicts I suppose.